core/src/ldml/C7043_ldml.md

```diff
@@ -339,11 +339,18 @@ There are `listCount` total lists.
 | ∆ | Bits | Name             | Description                                |
 |---|------|------------------|--------------------------------------------|
 | 0+|  32  | flags            | int: per-layers options                    |
-| 4+|  32  | hardware         | str: layout (`us`,`iso`,`jis`,`abnt2`)     |
+| 4+|  32  | hardware         | int: enumeration for hardware layout       |
 | 8+|  32  | layer            | int: index to first layer element          |
 |12+|  32  | count            | int: number of layer elements in this list |
 |16+|  32  | minDeviceWidth   | int: min device width in millimeters, or 0 |
 
+- `hardware`: an enumeration with the following values:
+  - 0: `none` (not valid for use with hardware)
+  - 1: `abnt2`
+  - 2: `iso`
+  - 3: `jis`
+  - 4: `us`
```

Comment:
Can we describe these here?

Reply:
Probably should refer to CLDR which also doesn't describe them. 

Do you have a recommendation ?

Reply:
How about: 

```suggestion
- `hardware`: an enumeration with the following values:
  - 0: `touch` - for touch layouts, not for use on hardware
  - 1: `abnt2` - Brazilian 103 key ABNT2 layout (iso + extra key left of right shift)
  - 2: `iso` - European 102 key layout (extra key right of left shift)
  - 3: `jis` - Japanese 109 key layout
  - 4: `us` - ANSI 101 key layout
```

But any text here is liable to generate controversial discussions... so our mileage may vary!

Reply:
Are you OK if i donate this text to TR35 instead?

Reply:
Sure; I wrote those descriptions very quickly so be prepared for discussion!